config/settings.yml

```diff
@@ -170,7 +165,6 @@
       - HOST_REGISTER_SUCCEEDED
       - IMPORTEXPORT_STARTING_EXPORT_TEMPLATE
       - IMPORTEXPORT_EXPORT_TEMPLATE
-      - ResourcePoolCreatedEvent
       - TemplateDeployedEvent
```

Comment:
Same thing there, this was added by d72199794cff3b7b82b6f61560a207423c38a5ec but isn't a VMware event that I can tell